salt/cdh/setup_hadoop.sls

```diff
@@ -18,17 +18,6 @@
 {% set app_packages_dir = pnda_home + "/app-packages" %}
 {% set pnda_graphite_host = salt['pnda.get_hosts_for_role']('graphite')[0] %}
 
-{%- set data_volume_list = [] %}
-{%- for n in range(flavor_cfg.data_volumes_count) -%}
```

Comment:
Remove data_volumes_count from the flavor cfg sls if no longer used (search for data_volumes_count)